docs/canvas/canvas-server-functions.asciidoc

```diff
@@ -9,191 +9,177 @@ the server when it hits one of these functions.
+|`string`
+|Comma separated list of meta fields, for example, `"_index,_type"`
 
-|count
-|number
-|The default is `1000`
+|`count`
+|`number`
+|The number of docs to pull back. Smaller numbers perform better 
+
+Default: `100`
 |===
 
-*Returns:* datatable
+*Returns:* `datatable`
 
 
 [float]
-=== pointseries
+[[essql_fn]]
+=== `essql`
 
-Turns a datatable into a point series model. Currently Canvas differentiates measure 
-from dimensions by looking for a <<canvas-tinymath-functions, Tiny Math function>>. If you enter a TinyMath 
-expression in your argument, Canvas treats that argument as a measure. Otherwise, it 
-is a dimension. Dimensions are combined to create unique keys. Measures are 
-then deduplicated by those keys using the specified TinyMath function.
+Specifies an {es} SQL.
 
-*Accepts:* datatable
+*Accepts:* `filter`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|x
-|string
-|The values along the X-axis
-
-|y
-|string
-|The values along the Y-axis
+|_Unnamed_ *** 
 
-|color
-|string
-|An expression to use in determining the mark's color
+Alias: `q`, `query`
+|`string`
+|An {es} SQL query
 
-|size
-|string
-|For elements that support it, the size of the marks
+|`count`
+|`number`
+|The number of documents to retrieve
 
-|text
-|string
-|For use in charts that support it, the text to show in the mark
+Default: `1000`
 |===
 
-*Returns:* pointseries
+*Returns:* `datatable`
 
 
 [float]
-=== server
-
-Forces the interpreter to return to the server.
-
-*Accepts:* Anything or nothing
+[[pointseries_fn]]
+=== `pointseries`
 
-*Returns:* Depends on your input and arguments
-&nbsp;
-
-[float]
-=== timelion
-
-Uses Timelion to extract one or more time series from many sources.
+Turns a `datatable` into a point series model. Currently Canvas differentiates measure 
+from dimensions by looking for a <<canvas-tinymath-functions, Tiny Math function>>. If you enter a TinyMath 
+expression in your argument, Canvas treats that argument as a measure. Otherwise, it 
+is a dimension. Dimensions are combined to create unique keys. Measures are 
+then deduplicated by those keys using the specified TinyMath function.
 
-*Accepts:* filter
+*Accepts:* `datatable`
 
 [cols="3*^<"]
 |===
 |Argument |Type |Description
 
-|_Unnamed_ (Aliases: `query`, `q`)
-|string
-|A Timelion query (default: `".es(*)"`)
+|`x`
+|`string`
+|The values along the X-axis
 
-|interval
-|string
-|Bucket interval for the time series (default: `auto`)
+|`y`
+|`string`
+|The values along the Y-axis
 
-|from
-|any
-|{es} date math string for the start of the time range (default: `now-1y`)
+|`color`
+|`string`
+|An expression to use in determining the mark's color
 
-|to
-|any
-|{es} date math string for the end of the time range (default: `now`)
+|`size`
+|`string`
+|For elements that support it, the size of the marks
 
-|timezone
-|any
-|Time zone for the time range (default: `UTC`)
+|`text`
+|`string`
+|For use in charts that support it, the text to show in the mark
```

Comment:
I'd make this language consistent with `177`.  Perhaps:

`For {things} that support it, the {property}`